src/main/java/com/jcabi/email/Protocol.java

```diff
@@ -97,6 +97,9 @@ public SMTP(final String hst, final int prt) {
 
     /**
      * SMTPS protocol.
+     * @todo #36:30min Write unit test for email sending with SMTPS protocol
+     *  using Greenmail mock server (see http://www.icegreen.com/greenmail/
+     *  for Greenmail docs and example code).
```

Comment:
@amihaiemil can you provide more context here? Why we need this test? Maybe link to the tickets with more info